### DISCUSSION TITLE
Looking at some website (Albania / Macedonia)

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#320] [Show and tell]
https://mw.bestvideostreaming.is/albapptv/?t=l_k (replace "stream" with "us" and delete token next to it)

http://us.bestvideostreaming.is/6263/playlist.m3u8 -> Kanal D Drama

http://us.bestvideostreaming.is/7711/playlist.m3u8 -> MFM
http://us.bestvideostreaming.is/7722/playlist.m3u8 -> AMOL

http://us.bestvideostreaming.is/8052/playlist.m3u8 -> RTSH Musikë

http://us.bestvideostreaming.is/8701/playlist.m3u8 -> RTSH Satelit
http://us.bestvideostreaming.is/8702/playlist.m3u8 -> RTK 1 HD
http://us.bestvideostreaming.is/8703/playlist.m3u8 -> V Plus ?
http://us.bestvideostreaming.is/8704/playlist.m3u8 -> Klan HD
http://us.bestvideostreaming.is/8705/playlist.m3u8 -> KTV HD
http://us.bestvideostreaming.is/8706/playlist.m3u8 -> Kosova
http://us.bestvideostreaming.is/8707/playlist.m3u8 -> Report TV HD
http://us.bestvideostreaming.is/8708/playlist.m3u8 -> Top Channel
http://us.bestvideostreaming.is/8709/playlist.m3u8 -> A TV

http://us.bestvideostreaming.is/8710/playlist.m3u8 -> K HD
http://us.bestvideostreaming.is/8711/playlist.m3u8 -> Albanian TV America
http://us.bestvideostreaming.is/8713/playlist.m3u8 -> Alsat
http://us.bestvideostreaming.is/8714/playlist.m3u8 -> RTVislam
http://us.bestvideostreaming.is/8715/playlist.m3u8 -> Oranews
http://us.bestvideostreaming.is/8716/playlist.m3u8 -> ?
http://us.bestvideostreaming.is/8717/playlist.m3u8 -> First Channel
http://us.bestvideostreaming.is/8718/playlist.m3u8 -> Elrodi TV
http://us.bestvideostreaming.is/8719/playlist.m3u8 -> EuroAL

http://us.bestvideostreaming.is/8720/playlist.m3u8 -> Klan Plus
http://us.bestvideostreaming.is/8721/playlist.m3u8 -> RTSH Shqip
http://us.bestvideostreaming.is/8723/playlist.m3u8 -> Opoja
http://us.bestvideostreaming.is/8724/playlist.m3u8 -> News24 AL
http://us.bestvideostreaming.is/8725/playlist.m3u8 -> National Geographic
http://us.bestvideostreaming.is/8726/playlist.m3u8 -> Digitalb Shkensë
http://us.bestvideostreaming.is/8727/playlist.m3u8 -> Digitalb Dramë
http://us.bestvideostreaming.is/8728/playlist.m3u8 -> MPT 2
http://us.bestvideostreaming.is/8729/playlist.m3u8 -> Digitalb Histori

http://us.bestvideostreaming.is/8730/playlist.m3u8 -> Digitalb Një HD (No Audio)
http://us.bestvideostreaming.is/8731/playlist.m3u8 -> Digitalb DY HD
http://us.bestvideostreaming.is/8735/playlist.m3u8 -> TV Plisi
http://us.bestvideostreaming.is/8737/playlist.m3u8 -> Bang Bang ALB
http://us.bestvideostreaming.is/8738/playlist.m3u8 -> Cufo
http://us.bestvideostreaming.is/8739/playlist.m3u8 -> Digitalb Natyra

http://us.bestvideostreaming.is/8742/playlist.m3u8 -> Digitalb Aktion HD
http://us.bestvideostreaming.is/8744/playlist.m3u8 -> RTSH Fêmijê
http://us.bestvideostreaming.is/8745/playlist.m3u8 -> ARTA HD
http://us.bestvideostreaming.is/8746/playlist.m3u8 -> Digitalb HD
http://us.bestvideostreaming.is/8747/playlist.m3u8 -> BabyTV
http://us.bestvideostreaming.is/8748/playlist.m3u8 -> 21 RTV
http://us.bestvideostreaming.is/8749/playlist.m3u8 -> 21 ?

http://us.bestvideostreaming.is/8750/playlist.m3u8 -> 21 Plus
http://us.bestvideostreaming.is/8751/playlist.m3u8 -> Digitalb ?
http://us.bestvideostreaming.is/8752/playlist.m3u8 -> Digitalb Komedi
http://us.bestvideostreaming.is/8753/playlist.m3u8 -> Digitalb Autor
http://us.bestvideostreaming.is/8754/playlist.m3u8 -> Digitalb ?
http://us.bestvideostreaming.is/8755/playlist.m3u8 -> Digitalb Hits HD
http://us.bestvideostreaming.is/8756/playlist.m3u8 -> 21 Mix
http://us.bestvideostreaming.is/8757/playlist.m3u8 -> Digitalb Thriller
http://us.bestvideostreaming.is/8758/playlist.m3u8 -> Digitalb Eurofilm
http://us.bestvideostreaming.is/8759/playlist.m3u8 -> Albania Folk

http://us.bestvideostreaming.is/8761/playlist.m3u8 -> Kanali 7
http://us.bestvideostreaming.is/8762/playlist.m3u8 -> Tring Shqip
http://us.bestvideostreaming.is/8763/playlist.m3u8 -> Tring World
http://us.bestvideostreaming.is/8764/playlist.m3u8 -> Tring Planet
http://us.bestvideostreaming.is/8765/playlist.m3u8 -> Tring History
http://us.bestvideostreaming.is/8766/playlist.m3u8 -> Tring Fantasy
http://us.bestvideostreaming.is/8767/playlist.m3u8 -> Tring Comedy
http://us.bestvideostreaming.is/8768/playlist.m3u8 -> Tring Life
http://us.bestvideostreaming.is/8769/playlist.m3u8 -> Tring Action HD

http://us.bestvideostreaming.is/8770/playlist.m3u8 -> Tring Super HD
http://us.bestvideostreaming.is/8771/playlist.m3u8 -> Living HD
http://us.bestvideostreaming.is/8771_6h/index.m3u8 -> Living HD Est
http://us.bestvideostreaming.is/8772/playlist.m3u8 -> U Smile
http://us.bestvideostreaming.is/8773/playlist.m3u8 -> Tring 3
http://us.bestvideostreaming.is/8774/playlist.m3u8 -> Tring Jolly HD
http://us.bestvideostreaming.is/8775/playlist.m3u8 -> Tring Tring
http://us.bestvideostreaming.is/8776/playlist.m3u8 -> Tip TV
http://us.bestvideostreaming.is/8777/playlist.m3u8 -> Tring Kids
http://us.bestvideostreaming.is/8778/playlist.m3u8 -> Muse
http://us.bestvideostreaming.is/8779/playlist.m3u8 -> ?

http://us.bestvideostreaming.is/8781/playlist.m3u8 -> Star HD
http://us.bestvideostreaming.is/8784/playlist.m3u8 -> Syril HD
http://us.bestvideostreaming.is/8785/playlist.m3u8 -> Q Sport News
http://us.bestvideostreaming.is/8786/playlist.m3u8 -> Pendimi TV HD
http://us.bestvideostreaming.is/8788/playlist.m3u8 -> RTV Dukagjini
http://us.bestvideostreaming.is/8789/playlist.m3u8 -> Klan Macedonia

http://us.bestvideostreaming.is/8790/playlist.m3u8 -> 21 TV Macedonia
http://us.bestvideostreaming.is/8792/playlist.m3u8 -> Peace TV
http://us.bestvideostreaming.is/8793/playlist.m3u8 -> dTV HD
http://us.bestvideostreaming.is/8794/playlist.m3u8 -> ALBUK
http://us.bestvideostreaming.is/8795/playlist.m3u8 -> BBF Music TV
http://us.bestvideostreaming.is/8796/playlist.m3u8 -> Tring Sport 1 HD (not to be added, might be flagged)
http://us.bestvideostreaming.is/8797/playlist.m3u8 -> Tring Sport 2 HD

http://us.bestvideostreaming.is/8802/playlist.m3u8 -> U Smile
http://us.bestvideostreaming.is/8803/playlist.m3u8 -> Digitalb Big Brother VIP 1
http://us.bestvideostreaming.is/8804/playlist.m3u8 -> Digitalb Big Brother VIP 2

http://us.bestvideostreaming.is/8817/playlist.m3u8 -> ?
http://us.bestvideostreaming.is/8818/playlist.m3u8 -> TV Dielli
http://us.bestvideostreaming.is/8819/playlist.m3u8 -> Technical issues on this one, see if it works later.

http://us.bestvideostreaming.is/8820/playlist.m3u8 -> Sofia HD
http://us.bestvideostreaming.is/8821/playlist.m3u8 -> AlbMusik HD
http://us.bestvideostreaming.is/8823/playlist.m3u8 -> TV Llapi

http://us.bestvideostreaming.is/8839/playlist.m3u8 -> Turbo TV

http://us.bestvideostreaming.is/8849/playlist.m3u8 -> ?

http://us.bestvideostreaming.is/8850/playlist.m3u8 -> Shenja
http://us.bestvideostreaming.is/8852/playlist.m3u8 -> RTV Fontana
http://us.bestvideostreaming.is/8853/playlist.m3u8 -> ?

http://us.bestvideostreaming.is/8880/playlist.m3u8 -> ?
http://us.bestvideostreaming.is/8881/playlist.m3u8 -> Komedi
http://us.bestvideostreaming.is/8882/playlist.m3u8 -> Max
http://us.bestvideostreaming.is/8883/playlist.m3u8 -> Drame
http://us.bestvideostreaming.is/8884/playlist.m3u8 -> Novela
http://us.bestvideostreaming.is/8886/playlist.m3u8 -> Animal Planet Albania
http://us.bestvideostreaming.is/8888/playlist.m3u8 -> Discovery Albania
http://us.bestvideostreaming.is/8889/playlist.m3u8 -> 7 HD

http://us.bestvideostreaming.is/8890/playlist.m3u8 -> Rita TV

http://us.bestvideostreaming.is/8902/playlist.m3u8 -> RTK 3 HD
http://us.bestvideostreaming.is/8903/playlist.m3u8 -> RTK 4 HD
http://us.bestvideostreaming.is/8904/playlist.m3u8 -> T7

**Comment by jkemp814** [comment]
VLC: Loaded playlist file inside GUI.
The first twelve played flawlessly. (8701-8713)
Then the next would not play. (8714-8853)
First, I thought I got banded. But I can go back and play the first twelve.

VLC: Loaded same playlist file on command line.
All channels play. 

MPV:
Played them all!

Summary: 100% streamable. Nice!

**Reply by BellezaEmporium** [reply]
More channels added.

**Reply by jkemp814** [reply]
All play well.

**Reply by ItzPanda05** [reply]
umm can u help me how to play them in vlc since I'm getting this error 
![image](https://user-images.githubusercontent.com/91161959/213312322-92999de2-65f5-425e-94d5-33b8f4cc8bb1.png)

**Reply by bleros** [reply]
Is not vlc problem you cant play this link anymore is region blocked

**Reply by ItzPanda05** [reply]
any way to fix that? i just wanna see Big Brother lol

**Reply by bleros** [reply]
No links are protected when they found that links are public then they added protection.
U can watch bb in kinemaja24.com or
https://bbalbaniavip.com/shiko-live/

**Comment by Agumonz98** [comment]
Welp, it seems the streams don't work anymore, time to update or renew the streams.

**Comment by Adr31233** [comment]
why this not work anymore? an u update

**Comment by bleros** [comment]
Hi do you have idea why is not working from my experience this error is that you are not allowed to watch to your country is geo blocked. Now i dont know which region belong this ?

403 Unauthorized

Streamer protection system doesn't allow you to watch this content.

**Comment by ItzPanda05** [comment]
the links dont work for me in VLC. Can someone help pls??

**Comment by BellezaEmporium** [comment]
It's a flussonic server and they added token protection. Won't work.

**Reply by bleros** [reply]
Yes
